s3prl/dataio/corpus/librispeech.py

```diff
@@ -86,6 +86,17 @@ def _parse_spk_to_gender(speaker_file: Path) -> dict:
 
 
 class LibriSpeech(Corpus):
```

Comment:
This change looks good to me, while I find that the `all_data` property not yet has doc. Could you follow the doc template in `fluent_speech_command` for librispeech? Thanks!

Reply:
Thanks Leo! I fixed it, please take a look